src/components/FakeCollectionStore.tsx

```diff
@@ -0,0 +1,30 @@
+import { createContext } from "react";
+import { CollectionItemModel } from "../domain/CollectionItemModel";
+import { CollectionModel } from "../domain/CollectionModel";
+
+class FakeCollectionStore {
+    public collections: CollectionModel[] = [];
+
+    public getCollections = (): CollectionModel[] => {
+        return this.collections;
+    }
+
+    public setCollections = (newCollections: CollectionModel[]): void => {
+        this.collections = newCollections;
+    }
+
+    public setFavorite = (favoritedItem: CollectionItemModel) => {
```

Comment:
Can a favorite get removed?

Reply:
Not with this current application, no. It will still cover our requirements though, given that there were none mentioning the removal of a favorite. Obviously, without a time crunch I'd feel better about figuring that out, but as we stand on requirements, we're in good shape.